.github/workflows/pr_request.yaml

```
@@ -0,0 +1,59 @@
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+
+      - name: Execute Gradle Shaded Build
+        run: ./gradlew shadowJar
+
+      - name: Cache artifacts
+        uses: actions/cache/save@v3
+        with:
+          path: |
+            build/libs
+          key: ${{ steps.commit.outputs.short }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)